kairon/api/app/routers/bot/channels.py

```
@@ -254,3 +254,14 @@ async def get_channel_metrics(
     Get Channel metrics (Failures/Successes).
     """
     return Response(data=MessageBroadcastProcessor.get_channel_metrics(channel_type, current_user.get_bot()))
+
+@router.get("/media/upload/{bsp_type}/{media_id}", response_model=Response)
+async def bsp_upload_media(
+    media_id: str = Path(description="Id of the document"),
+    access_token: str = Query(description="360Dialog access token"),
+    bsp_type: str = Path(description="Business service provider type", examples=[WhatsappBSPTypes.bsp_360dialog.value]),
+    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS)
```

Ruff (0.8.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)